<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>more_window</class>
 <widget class="QWidget" name="more_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_7">
   <item row="0" column="0" rowspan="2">
    <layout class="QVBoxLayout" name="menu_verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>菜单</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QPushButton" name="eol_test_pushButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>EOL调试</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QPushButton" name="update_pushButton_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>固件升级</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="receive_data_groupBox">
     <property name="title">
      <string>数据接收</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QLabel" name="rx_frame_cnt_label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>RX帧数：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="rx_byte_cnt_label">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>RX字节数：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="tx_frame_cnt_label">
                <property name="text">
                 <string>TX帧数：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="tx_byte_cnt_label">
                <property name="text">
                 <string>TX字节数：</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_3">
              <item>
               <widget class="QLabel" name="rx_frame_num_label">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="rx_byte_num_label">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="tx_frame_num_label">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="tx_byte_num_label">
                <property name="text">
                 <string>0</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="display_mask_label">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>显示CAN ID掩码(0x)：</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="display_mask_lineEdit">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>FFFF</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="display_direction_checkBox_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>显示收发</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="display_time_stamp_checkBox">
            <property name="text">
             <string>显示时间</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="auto_wrap_checkBox_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>自动换行</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="mask_en_checkBox">
            <property name="text">
             <string>掩码使能</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QComboBox" name="display_ch_comboBox"/>
            </item>
            <item>
             <widget class="QCheckBox" name="display_str_checkBox">
              <property name="text">
               <string>显示字符</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="clear_pushButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>清除接收</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTextBrowser" name="ch1_receive_data_textBrowser"/>
          </item>
          <item>
           <widget class="QTextEdit" name="ch1_receive_data_textEdit"/>
          </item>
          <item>
           <widget class="QTextEdit" name="ch2_receive_data_textEdit"/>
          </item>
          <item>
           <widget class="QTextBrowser" name="ch2_receive_data_textBrowser"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="send_data_groupBox">
     <property name="title">
      <string>数据发送</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QGroupBox" name="send_par_groupBox">
          <property name="title">
           <string>发送设置</string>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QComboBox" name="channel_num_comboBox">
             <item>
              <property name="text">
               <string>通道0</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>通道1</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="id_label">
             <property name="text">
              <string>ID（0x）:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="id_lineEdit"/>
           </item>
           <item>
            <widget class="QLabel" name="frame_type_label">
             <property name="text">
              <string>帧类型：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="frame_type_comboBox">
             <item>
              <property name="text">
               <string>标准帧</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>扩展帧</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="protocol_label">
             <property name="text">
              <string>协议：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="protocol_comboBox">
             <item>
              <property name="text">
               <string>CAN</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>CANFD</string>
              </property>
             </item>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="canfd_pluse_label">
             <property name="text">
              <string>CANFD加速：</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="canfd_pluse_comboBox">
             <item>
              <property name="text">
               <string>否</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>是</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="data_label">
          <property name="text">
           <string>数据：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="data_lineEdit">
          <property name="text">
           <string>00 01 02 03 04 05 06 07 08 09 10 11 12 13 14 15 16 17 18 19 20 21 22 23 24 25 26 27 28 29 30 31 32 33 34 35 36 37 38 39 40 41 42 43 44 45 46 47 48 49 50 51 52 53 54 55 56 57 58 59 60 61 62 63</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="timer_horizontalLayout_2">
        <item>
         <widget class="QLabel" name="period_label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>周期ms：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="period_lineEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="timer_checkBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>定时发送</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="send_pushButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>发送一帧</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
